build-scripts/generateConfigurationAttributes.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/* tslint:disable */
+
+import * as fs from 'fs'
+// tslint:disable-next-line:no-implicit-dependencies
+import { JSONSchema4 } from 'json-schema'
+import { compile } from 'json-schema-to-typescript'
+import * as packageJson from '../package.json'
+
+const config = [
+    {
+        debugger: 'aws-sam',
+        requestType: 'direct-invoke' as 'direct-invoke',
+        outputFile: 'src/shared/sam/debugger/awsSamDebugConfiguration.gen.ts',
+        imports: ["import * as vscode from 'vscode'"],
+        topLevelClass: 'AwsSamDebuggerConfiguration'
+    }
+]
+
+const header = `
+/* tslint:disable */
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
```

Comment:
Nit: this only ever reads the package.json file, so we should explicitly mention it. Also, we should mention the npm command instead of running json-schema-to-typescript.

Reply:
I literally copied this from what gets auto-generated so that I could add additional imports ;)